loader/CMakeLists.txt

```diff
@@ -388,10 +326,14 @@ else()
         )
 # cmake-format: on
     endif()
+endif()
 
-    if(NOT APPLE)
-        target_compile_definitions(vulkan PRIVATE _XOPEN_SOURCE=500) # hush compiler warnings for readlink
```

Comment:
Is this no longer needed?

Reply:
I haven't seen the error when compiling. I believe the error was that one of the targets (asm_offset I think) didn't have _GNU_SOURCE and included `vk_loader_platform.h`, thus would cause this compiler error/warning to occur.